### DISCUSSION TITLE
¿Texture problem in android 10?

### DISCUSSION THREAD

**Discussion by Helloooo908** [skyline-emu/skyline#1570] [Q&A]
I have a problem with cuphead and graveryard, the colors are inverted or invisible, I'm on android 10 with SD865. But I would like to know if it opens a solution in the future or not, please I await your response. 🙏🙏
![Screenshot_2022-06-16-09-02-38-368_skyline emu](https://user-images.githubusercontent.com/107641522/174130642-fcfed5f5-3a5d-4bb6-9ab3-868a6242ec42.jpg)

**Comment by 13gkd** [comment]
> same problem in my device

**Reply by Helloooo908** [reply]
> > mismo problema en mi dispositivo

And I think there will be no solution, I created a thread in the problem and they closed it for me because they do not have in mind to solve that problem. I already have to look for a custom rom that has the updated gpu drivers.

**Reply by yhak22** [reply]
I have sd865 and it didn't happened to me.

**Reply by hope3x** [reply]
> > > mismo problema en mi dispositivo
> 
> And I think there will be no solution, I created a thread in the problem and they closed it for me because they do not have in mind to solve that problem. I already have to look for a custom rom that has the updated gpu drivers.

You have Mali?

**Reply by Helloooo908** [reply]
> Yo tengo sd865 y no me paso.

If this does not happen to you, it is because you must have android 11 and therefore updated the gpu drivers.

**Reply by Helloooo908** [reply]
> > > > mismo problema en mi dispositivo
> > 
> > 
> > Y creo que no habrá solución, creé un hilo en el problema y me lo cerraron porque no tienen en mente solucionar ese problema. Ya tengo que buscar una rom personalizada que tenga los controladores gpu actualizados.
> 
> ¿Tienes Malí?

No bro I have snapdragon 865 on android 10, I don't update it for fear, since my mobile is 2 years old and the performance is going to kill me. And in android 10 it works like a bullet and it doesn't even get hot.

**Reply by Helloooo908** [reply]
Excuse me for any words, I use google translate

**Reply by yhak22** [reply]
maybe u have old driver than mine , i have v502 if u have older i think u need upgrade driver.
